test/prototype_transforms_dispatcher_infos.py

```diff
@@ -426,4 +426,13 @@ def xfail_all_tests(*, reason, condition):
             skip_dispatch_feature,
         ],
     ),
+    DispatcherInfo(
+        F.uniform_temporal_subsample,
+        kernels={
+            features.Video: F.uniform_temporal_subsample_video,
+        },
+        test_marks=[
+            skip_dispatch_feature,
+        ],
+    ),
```

Comment:
After the `KernelInfo` is added, adding the corresponding `DispatcherInfo` is much simpler. We only need to define which kernel will be dispatched to for which feature type and the inputs will be automatically picked up from the corresponding `KernelInfo`'s. In our case we only need add one predefined `TestMark` to skip the test checking if the dispatch to the corresponding method on the feature class works, since this dispatcher does not have such a method.